File: ce/snippets/visualbasic/CRMV8/businessmanagementvb/vb/locksalesorderpricing.vb
' <snippetlocksalesorderpricing>


 ' Relock the order pricing
 Dim lockOrderRequest As New LockSalesOrderPricingRequest() With
     {
         .SalesOrderId = _salesOrderId
     }
 _serviceProxy.Execute(lockOrderRequest)

' </snippetlocksalesorderpricing>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/businessmanagement/cs/locksalesorderpricing.cs
// <snippetlocksalesorderpricing>


 // Relock the order pricing
 LockSalesOrderPricingRequest lockOrderRequest =
     new LockSalesOrderPricingRequest()
     {
         SalesOrderId = _salesOrderId
     };
 _serviceProxy.Execute(lockOrderRequest);

// </snippetlocksalesorderpricing>